Xamarin.Forms.Xaml/ApplyPropertiesVisitor.cs

```diff
@@ -290,148 +267,204 @@ static string GetContentPropertyName(IEnumerable<CustomAttributeData> attributes
 			return null;
 		}
 
-		public static void SetPropertyValue(object xamlelement, XmlName propertyName, object value, object rootElement,
-			INode node, HydratationContext context, IXmlLineInfo lineInfo)
+		public static void SetPropertyValue(object xamlelement, XmlName propertyName, object value, object rootElement, INode node, HydratationContext context, IXmlLineInfo lineInfo)
```

Comment:
It starts here.

`SetPropertyValue` was refactored from ~150LOCs down to ~30 including lots of comments. It's now easier to read, understand and maintain.

I'll add a comment when the changes are actually new, and not refactorings